algorithms/src/Kokkos_Sort.hpp

```diff
@@ -628,6 +636,30 @@ std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
   bin_sort.sort(exec, view);
 }
 
+#if defined(KOKKOS_ENABLE_ONEDPL)
+template <class DataType, class... Properties>
+void sort(const Experimental::SYCL& space,
+          const Kokkos::View<DataType, Properties...>& view,
+          bool const = false) {
```

Comment:
The discussion related to https://github.com/kokkos/kokkos/pull/5372 also applies here.

Reply:
Resolved.